images/process/imagecollector/entrypoint.bash

```diff
@@ -36,8 +36,9 @@ fi
 TZ="Europe/Berlin" date > lastScan
 git add lastScan || true
 
-echo "Old csv is not to be used anymore" # TODO: Remove
-git rm imagesAndCluster.csv || true
+echo "Old json is not to be used anymore" # TODO: Remove
```

Comment:
The collector is deprecated due to the go one.
This has nothing to do with the other changes in this PR.